src/nbl/video/IGPUCommandBuffer.cpp

```diff
@@ -1774,6 +1888,117 @@ bool IGPUCommandBuffer::resolveImage(const IGPUImage* const srcImage, const IGPU
     return resolveImage_impl(srcImage, srcImageLayout, dstImage, dstImageLayout, regionCount, pRegions);
 }
 
+bool IGPUCommandBuffer::setRayTracingPipelineStackSize(uint32_t pipelineStackSize)
+{
+    return setRayTracingPipelineStackSize_impl(pipelineStackSize);
```

Comment:
you need to check if the bound pipeline was created with `cachedParams.dynamicStackSize`  (basically `VK_DYNAMIC_STATE_RAY_TRACING_PIPELINE_STACK_SIZE_KHR`)

Reply:
also it would be nice to track that its been called since commandbuffer begin (which will reset it) to check `VUID-vkCmdTraceRaysKHR-None-09458` and `VUID-vkCmdTraceRaysIndirectKHR-None-09458`

Reply:
you're only allowed to set `m_haveRtPipelineStackSize = true` and not return false if a pipeline with dynamic stack size is already bound (checked the Vulkan spec)